### DISCUSSION TITLE
Suggestion: Claims for sale (not claim blocks)

### DISCUSSION THREAD

**Discussion by Borbarad13** [GriefPrevention/GriefPrevention#521] [Ideas]
Hello everyone,

after I subdivide a claim (city), it seems (as a major / admin) that I have to manually give other people trust to to these subdivisions. What would be really great though, if I can mark these plots as for "sale" and that players can buy them and start building on them -without me being online-.

Is there any way to this? If not I really would love to see such a feature implemented.

**Comment by TomLewis** [comment]
Use areashop instead. Automate it all.

**Comment by Borbarad13** [comment]
Hey FrozenBeard,
I looked at the plugin and I am not a fan of it, since it comes with a lot of dependencies (e.g. worldEdit) and unnecessary stuff.

If there would just be a command to grant trust that is runnable on a command block, then I could create a function that checks for the player's balance and grants trust. I could place these blocks in all the plots and players could buy the land for themselves. If existing trust commands were runnable by a commandblock (of course then they need an extra parameter on whose behalf the command is executed), it would be totally possible to automate things with functions/command blocks without the plugin requiring the admin to be on.

What do you think?

**Comment by RoboMWM** [comment]
GPRealEstate? @nullcase might know some stuff on this.

**Comment by Borbarad13** [comment]
I saw GPRealEstate before, but it was old/not maintained/unsupported, but I found newer repositories:
- https://github.com/RoboMWM/GPRealEstate (this one even by you)
- https://github.com/SuperPyroManiac/GPRealEstate
So I guess I could give this a try. Thanks!

**Comment by RoboMWM** [comment]
Nullcase requested me to decompile and fix (a year before the original author found his code and republished) so you may want to use mine if that's what you're looking for. But Nullcase has other features in another one hence why he probably knows the status on this type of addon better than I do.
